src/todo/todo.lua

```diff
@@ -92,6 +92,10 @@ function todo.update(element, index)
     else
         original.assignee = nil
     end
+    -- Set the last updater
+    local players, _ = todo.get_player_list()
+    original.by = players[2]
```

Comment:
Is this always true? Even for larger multiplayer maps?

Reply:
After thinking about this it should not be true. The ordering of the players list is the ordering in which the players joined. It is actually the same for everyone.

Reply:
I changed the way to get the player's name. I had to pass the player object to the create/update functions but the player index is only stored in the event...